spec/adjacent_spec.rb

```diff
@@ -0,0 +1,176 @@
+require File.expand_path('watirspec/spec_helper', File.dirname(__FILE__))
+
+describe "Adjacent Elements" do
```

Comment:
It will still be in `spec/watirspec` directory. Knowing that you already have PR to watirspec, let's just move the file there.